NASA-AMMOS/aerie-ui
# Replace View Editor with View Upload Modal

This resolves https://github.com/NASA-AMMOS/aerie-ui/issues/382

To test:
1. Navigate into a plan
2. Hover over the View menu
3. Select "Upload view file"
4. Verify an upload modal opens
5. Fill in view name
6. Choose a view file to upload (`e2e-tests/data` contains a valid and invalid test file)
7. Verify that after clicking the "Upload View" button that the view is updated and the name reflected on the View menu
